Improve UX without workspace

shamshaev opened the pull request:
Centered block with short description was added. Adaptive layout, both languages are supported. Here is the link to the deployed app (https://hexlet-correction-vvt2.onrender.com).

![Screenshot from 2024-09-22 19-08-27](https://github.com/user-attachments/assets/914c2d85-12d4-4f2b-8e93-19544959c5ca)

@fey I also added button "Learn more". Now It's inactive. My proposal is to link this button later to instructions which will be made in issue #81 by @alt-schince-technology. Or you can just remove this button if you decide another way to link those instructions.

fey commented:
@shamshaev yeah looks good (saw screenshot).
You can remove Learn more button, we ll add it later.

shamshaev commented:
@fey I removed button "Learn more". Here is the screenshot. Also updated on Render, the link is same.

![Screenshot from 2024-09-23 22-14-50](https://github.com/user-attachments/assets/2d173ab9-73ef-424c-90f2-64dc8d21e34f)

fey commented:
lgtm
@Malcom1986 can you review? =)
